docs/articles/documentation/reference/test-api/testcontroller/switchtoiframe.md

```diff
@@ -41,6 +41,25 @@ test('switching to an iframe', async t => {
 
 You can switch to the main window from the `<iframe>` with the [t.switchToMainWindow](switchtomainwindow.md) method.
 
+## Wait Until an \<iframe\> Is Loaded
+
+TestCafe implements [wait mechanisms](../../../guides/concepts/built-in-wait-mechanisms.md) that automatically suspend the test until all required page elements are loaded and ready for interaction. These mechanisms apply to page loads, animations, XHR requests, iframe initializations, etc. TestCafe waits until the target element is ready, or reports an error if this does not happen within a timeout.
+
+This example shows how to allow more time for an `iframe` to load if the default timeout is not enough.
```

Comment:
`if the [default timeout](https://devexpress.github.io/testcafe/documentation/reference/command-line-interface.html#--selector-timeout-ms) is not enough`